bot/cogs/source.py

```diff
@@ -32,6 +27,14 @@ async def convert(self, ctx: commands.Context, argument: str) -> SourceType:
         if cmd:
             return cmd
 
+        tags_cog = ctx.bot.get_cog("Tags")
+
+        if not tags_cog:
+            await ctx.send("Unable to get `Tags` cog.")
+            return commands.ExtensionNotLoaded("bot.cogs.tags")
+        elif argument.lower() in tags_cog._cache:
+            return argument.lower()
```

Comment:
Returning an exception rather than raising it is weird. I don't agree with this approach. I think that if the cog isn't loaded, it should not attempt to get the tag and let the `BadArgument` on like 38 be raised.

Reply:
Resolved in 51d6816.